packages/zendesk-adapter/src/change_validators/organization_existence.ts

```diff
@@ -85,11 +86,15 @@ export const organizationExistenceValidator: (client: ZendeskClient, fetchConfig
 
           // If the instance includes an organization that does not exist, we won't allow a change to that instance
           if (nonExistingOrgs.size > 0) {
+            // Check if any of the non-existing orgs are numbers, if so, we should recommend the user to resolve them
+            const nonExistingOrgsAreIds = !orgIdsResolved || wu(nonExistingOrgs.values()).some(org => isNumber(org))
+            const resolveOrgsRecommendation = nonExistingOrgsAreIds ? '. Salto can identify organizations by their names. This requires setting the \'resolveOrganizationIDs\' to true in the zendesk configuration file of both source and target envs and fetch.\n'
+                + 'More information about Salto\'s config files can be found here' : ''
```

Comment:
1. readability nit:
```suggestion
            const resolveOrgsRecommendation = nonExistingOrgsAreIds
              ? '. Salto can identify organizations by their names. This requires setting the \'resolveOrganizationIDs\' to true in the zendesk configuration file of both source and target envs and fetch.\n'
                + 'More information about Salto\'s config files can be found here'
              : ''
```
2. what is "here" supposed to signify? are we adding a link?